edenia/antelope-tools
# Feat/improve UI nodes page 864

### Display producer nodes horizontally

### What does this PR do?

- Resolve #864
- Resolve #653
- Resolve #976  
- Display nodes by producer using the new db model to show endpoint health
- Open the endpoint tooltip when the mouse is over the health indicator
- Remove unnecessary files
- Remove box from ProducerHealthIndicators component
- Modify object features with list and keys for lacchain network

### Screenshots

![imagen](https://user-images.githubusercontent.com/66583677/196792318-b7bf9e51-3d58-487c-8a08-8c6101358cc5.png)
![imagen](https://user-images.githubusercontent.com/66583677/196554720-279cbc7d-7e47-4111-9a68-d4db10344078.png)
![imagen](https://user-images.githubusercontent.com/66583677/196792604-1938516c-801b-4b45-98f1-0f7c6004ddb4.png)


### Steps to test

1. Run the project locally
2. Go to nodes pages
3. Check the change
